source/governance/committee.rst

```diff
@@ -288,55 +335,69 @@ facilitating meetings and serving as a focal point for the
 Steering Committee.
 
-Representation The CGNS Steering Committee will be made up of
-representatives from specific institutions, rather than
-individuals. Changes or additions to Steering Committee membership
-will be based on potential contribution to the standard. Membership on
+.. _org_Representation-ref:
+
+Representation
+^^^^^^^^^^^^^^
+The CGNS Steering Committee will consist of representatives from designated
+institutions rather than individual members. Changes or additions to Steering
+Committee membership will be based on potential contributions to the standard. Membership in
 the Steering Committee will be limited to 30 institutions that
-actively participate in the development, maintenance, distribution and
-use of the CGNS Standard. No more than 5 institutions may be related,
-i.e., have the same parent organization. Changes to the Membership
+actively participate in developing, maintaining, distributing, and using
+the CGNS Standard. No more than five institutions,
+i.e., have the same parent organization, may be related. Changes to the Membership
 (including the limit on the number of institutions) will be determined
-by consensus, or if required, a two-thirds majority of the existing
+by consensus or, if required, a two-thirds majority of the existing
 Membership.
 
-To help satisfy the duties of the Steering Committee as a whole, the
-minimal responsibilities of each individual Steering Committee member
+To help satisfy the :ref:`duties of the Steering Committee as a whole <mission_sec-ref>`,
+the minimal responsibilities of each Steering Committee member
 are to:
 
-Attend as many telecons/meetings as possible, but not less than one
-per year Read and send comments on proposals or other issues when
-asked to do so Vote when asked to do so More active participation -
-including support, software development, and actively working to
-improve and promote CGNS - is encouraged.  Standing Committees The
-CGNS Steering Committee may constitute Standing Committees, in an
+- Attend as many telecons/meetings as possible, but not less than one per year
+- Read and send comments on proposals or other issues when asked to do so
+- Vote when asked to do so
+
+More active participation -- including support, software development, and actively working to
+improve and promote CGNS -- is encouraged.
+
+.. _org_Standing_Committees-ref:
+
+Standing Committees
+^^^^^^^^^^^^^^^^^^^
+
+The CGNS Steering Committee may constitute Standing Committees in an
 ongoing or temporary basis, to which it may delegate various
 responsibilities. The Standing Committees will report and make
 recommendations to the Steering Committee who will retain the
 authority to act and make final decisions.
 
-Software and Documentation Support Team The CGNS Steering Committee
-will be responsible for selecting one or more organizations to
+.. _org_Software_and_Documentation-ref:
+
+Software and Documentation Support Team
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
Switched to man select an organization in case one can not be found, and the actions to if one is not specified.